RefRed/plot/single_click_plot.py

```diff
@@ -1,13 +1,14 @@
 import time
-
-import RefRed.constants
 from RefRed.plot.popup_plot_1d import PopupPlot1d
 from RefRed.plot.popup_plot_2d import PopupPlot2d
 from RefRed.plot.launch_stitching_manual_axis import LaunchStitchingManualXAxis
 from RefRed.plot.launch_stitching_manual_axis import LaunchStitchingManualYAxis
 from RefRed.gui_handling.gui_utility import GuiUtility
 
 
+DOUBLE_CLICK_DELAY = 0.4  # seconds
```

Comment:
same, looks like a lot of hard-coded constants which may be better managed if in one place rather that spread-out over the code.